### ISSUE TITLE
action does not archive all installed files

### ISSUE THREAD

**Issue by kaz-utashiro** [office-tecoli/actions-use-apt-tools#1]
dpkg -L does not provide complete file lists sometimes.

https://github.com/kaz-utashiro/sdif-tools/runs/5752142869

https://packages.ubuntu.com/impish/mecab-ipadic-utf8

```
# dpkg -L mecab-ipadic-utf8
/.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/mecab-ipadic-utf8
/usr/share/doc/mecab-ipadic-utf8/copyright
/usr/share/lintian
/usr/share/lintian/overrides
/usr/share/lintian/overrides/mecab-ipadic-utf8
/var
/var/lib
/var/lib/mecab
/var/lib/mecab/dic
/var/lib/mecab/dic/ipadic-utf8
/usr/share/doc/mecab-ipadic-utf8/changelog.Debian.gz
```

**Comment by kaz-utashiro** [comment]
mecab-ipadic package generate files under /var/lib during installation, so they are not included in the package files.

https://twitter.com/knok/status/1509333234685181953

**Comment by kaz-utashiro** [comment]
fixed in v0.5 by introducing *method* parameter.
